daml-assistant/daml-project-config/DAML/Project/Util.hs

```diff
@@ -3,10 +3,25 @@
 
 module DAML.Project.Util where
```

Comment:
I love export lists...

Reply:
Can we hlint for that?

Reply:
Yes! If there's broad support I'll do it. You just add `- warn: {name: Use explicit module export list}` in the `.hlint.yaml`

Reply:
I'm in favor!

Reply:
:+1: